src/LibraryManager.Vsix/Commands/CommandTable/VSCommandTable.en.vsct

```diff
@@ -31,15 +31,15 @@
         <CommandFlag>DefaultInvisible</CommandFlag>
         <CommandFlag>DynamicVisibility</CommandFlag>
         <Strings>
-          <ButtonText>Restore Client-Side Libraries</ButtonText>
+          <ButtonText>Restore Client-Side &amp;Libraries</ButtonText>
```

Comment:
>Libraries [](start = 47, length = 9)

Seems weird to pick the 'L' as the mnemonic here, I would have thought a character from Restore would be more appropriate.

Reply:
I agree. I would have used "R", except that "Properties" uses "r" and is available on every item, so would cause a conflict.
"C" is also heavily used. (Though, strangely, not for "Copy".)

Reply:
All letters of "Restore" are already being used by:
- (R) Properties
- (E) Enable Restore Client-Side Libraries on Build (my new feature)
- (S) Scope to this
- (T) Cut
- (O) Open

"C" is also being used by "Clean Client-Side Libraries".

So "L" seems the best choice.